docs/local_development.md

```diff
@@ -45,49 +43,140 @@ yarn install --force --frozen-lockfile
+execution, look there.
 
-The Daml Sandbox will automatically be bootstrapped with contracts from `daml/Setup.daml`.
+You may run multiple instances of the same trigger with different parties - each instance will get its own log.
 
-If you would like to change which Daml Script file is run when the Daml Sandbox starts, you can modify the `init-script` field in `daml.yaml`.
+### Stop the Triggers
 
-To access the frontend, point your browser to `http://localhost:3000`. To view the Daml ledger state in the Daml Navigator, point your browser to `http://localhost:7500`.
+You may explicitly stop any trigger by calling its symmetric `stop-*` make target on it. For example
+
+```sh
+$ make stop-clearing-trigger party=AnyPartyName
+$ make stop-matching-engine party=AnyPartyName
+$ make stop-settlement-trigger party=AnyPartyName
+$ make stop-autoapprove party=AnyPartyName
+```
 
-To stop running background processes:
+To stop all the running auto approve triggers in one go, run
 
+```sh
+$ make stop-autoapprove-all
 ```
-# stop the Daml server
-make stop_daml_server
 
-# stop all bots
-make stop_bots
+Be aware that stopping a trigger will delete the log file associated with it.
+
+Additionally, the trigger processes will die if the main sandbox process is killed,
+which may leave behind lingering files in `.dev`.
+
+### Start the UI
+
+The UI can be run locally through the webpack dev server. Simply
 
-# if you are using the included matching engine:
-make stop_matching_engine
+```sh
+$ cd ui
+$ yarn start
 ```
 
-Alternatively to running the Daml server in the background, `daml start` can be run in a separate terminal window.
+This should result in a UI on `localhost:3000`. Locally, files are watched for changes and the server
+should reload automatically.
+
+## Exberry Integration
+
+The Exberry integration can be started up locally. There are two components: the adapter, included
+in this repository, and the integration itself which is in its own repository.
+
+1. Clone [`daml-dit-integration-exberry`](https://github.com/digital-asset/daml-dit-integration-exberry) in a sibling directory next to `da-marketplace/`
+2. Open two terminals, one in the `da-marketplace/` directory, and one in the `daml-dit-integration-exberry/` directory.
+3. In `daml-dit-integration-exberry/`, create a file in the repo root named `int_args.yaml`. Fill it with the template below, replaced with your Exberry account details.
+
+```yaml
+"metadata":
+  "runAs": "Exchange"
+  "username": "EXBERRY_USERNAME"
+  "password": "EXBERRY_PASSWORD"
+  "clientId": "EXBERRY_CLIENTID"
+  "tradingApiUrl": "EXBERRY_TRADING_API_URL"
+  "adminApiUrl": "EXBERRY_ADMIN_API_URL"
+  "tokenUrl": "EXBERRY_TOKEN_URL"
+  "apiKey": "EXBERRY_API_KEY"
+  "secret": "EXBERRY_SECRET"
+```
+
+4. In `daml-dit-integration-exberry/`, run `make all`
+5. In `da-marketplace/`, run `daml start --sandbox-option ../daml-dit-integration-exberry/dabl-integration-exberry-0.7.3.dar`
```

Comment:
I wonder if we should do `./dabl-integration-exberry-{VERSION}.dar` same for the line below so we don't have to update this README every time the integration has a new version.

Reply:
good idea